unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -341,28 +341,27 @@ public void ChangeEditModeStatusByShortcut()
         if (isEnteringEditMode)
             return;
 
+        if (isBuilderInWorldActivated)
+        {
+            HUDController.i.builderInWorldMainHud.ExitStart();
+            return;
+        }
+
         FindSceneToEdit();
 
         if (!UserHasPermissionOnParcelScene(sceneToEdit))
         {
             ShowGenericNotification(BuilderInWorldSettings.LAND_EDITION_NOT_ALLOWED_BY_PERMISSIONS_MESSAGE);
             return;
         }
-        else if (IsParcelSceneDeployedFromSDK(sceneToEdit))
+        if (IsParcelSceneDeployedFromSDK(sceneToEdit))
```

Comment:
Why did you remove the `else` here? In case the user don't have pemissions to edit a land and also this scene is created from the SDF, do we want to show both notifications at the same time? Would it not a bit weird?

Reply:
The else here is redundant, if the user doesn't have permission on the parcel scene we will return and we will never reach that part of the code.  So the else is not needed since the user doesn't have permission on the parcel scene it will never check if it is from SDK. And if the user has permission it will always check if it is from the SDK

I really don't mind putting it back

Reply:
Oh I didn't see this `return;`, perfect then! Thanks!